xml/admin_ceph_upgrade.xml

```diff
@@ -325,6 +327,29 @@ mds standby for name = mds.456-another-mds
      &cephfs; load.
     </para>
    </listitem>
+   <listitem>
+    <para>
+     <literal>osd_scrub_max_interval</literal> and
+     <literal>osd_scrub_max_interval</literal> settings are used by both OSD
+     and MON daemons. OSDs use these settings to decide when to run scrub, and
+     MONs use them to decide if a warning about scrub not running in-time
+     (running too long) should be show. Therefore if non-default settings are
+     used they should be visible by both OSD and MON daemons (i.e. defined
+     either in both <literal>[osd]</literal> and <literal>[mon]</literal>
+     sections, or in the <literal>[global]</literal> section), otherwise the
+     monitor may give a false alarm.
+    </para>
+    <para>
+     In SES 5.5 the monitor warning are disabled by default and the issue may
+     not be noticed if the settings are overridden in the
+     <literal>[osd]</literal> section only. But when the monitors are upgraded
+     to SES 6, it will start to complain, because the warnings are enabled here
+     by default. So if you have non-default scrub settings in your
+     configuration defined in the <literal>[osd]</literal> section only, it is
```

Comment:
```suggestion
     <literal>[osd]</literal> section only. But when the monitors are upgraded
     to SES 6, it will start to complain, because in this version the warnings are
     enabled by default. So if you define non-default scrub settings in your
     configuration only in the <literal>[osd]</literal> section, it is
```